contracts/.env.localhost

```diff
@@ -21,3 +21,7 @@ RIVERSCAN_URL=
82815181106107b2576107b2610c7d565b6020026020010151602001515b1561081357818383815181106107d7576107d7610c7d565b60200260200101516040517f3b4c7a5f0000000000000000000000000000000000000000000000000000000081526004016103c0929190610cc8565b508061081e81610ce9565b9150506105f4565b50915091565b803573ffffffffffffffffffffffffffffffffffffffff8116811461085057600080fd5b919050565b60006020828403121561086757600080fd5b6108708261082c565b9392505050565b60006020828403121561088957600080fd5b5035919050565b600080604083850312156108a357600080fd5b6108ac8361082c565b946020939093013593505050565b60005b838110156108d55781810151838201526020016108bd565b50506000910152565b600081518084526108f68160208601602086016108ba565b601f017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0169290920160200192915050565b60208152600061087060208301846108de565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b60405160c0810167ffffffffffffffff8111828210171561098d5761098d61093b565b60405290565b604051601f82017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe016810167ffffffffffffffff811182821017156109da576109da61093b565b604052919050565b8035801515811461085057600080fd5b60006020808385031215610a0557600080fd5b823567ffffffffffffffff80821115610a1d57600080fd5b818501915085601f830112610a3157600080fd5b813581811115610a4357610a4361093b565b8060051b610a52858201610993565b9182528381018501918581019089841115610a6c57600080fd5b86860192505b83831015610b9f57823585811115610a8a5760008081fd5b860160c07fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0828d038101821315610ac15760008081fd5b610ac961096a565b610ad48b85016109e2565b81526040610ae38186016109e2565b8c830152606080860135828401526080610afe81880161082c565b8285015260a091508187013581850152508486013594508a851115610b235760008081fd5b84860195508f603f870112610b3a57600094508485fd5b8c86013594508a851115610b5057610b5061093b565b610b608d85601f88011601610993565b93508484528f82868801011115610b775760008081fd5b848287018e86013760009484018d019490945250918201528352509186019190860190610a72565b9998505050505050505050565b604080825283519082018190526000906020906060840190828701845b82811015610be7578151151584529284019290840190600101610bc9565b50505083810382850152845180825282820190600581901b8301840187850160005b83811015610c55577fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0868403018552610c438383516108de565b94870194925090860190600101610c09565b50909998505050505050505050565b600060208284031215610c7657600080fd5b5051919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b60008251610cbe8184602087016108ba565b9190910192915050565b828152604060208201526000610ce160408301846108de565b949350505050565b60007fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff8203610d41577f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b506001019056fea264697066735822122049ef042991a3ec1b25eb7d0bd2d2faaceec6986d39c10052cc3d76bbe0bb4ad664736f6c63430008120033
```

Comment:
Not sure I'm happy saving this bytecode into environment variables, we have our own MultiCall.sol contract, I would rather use that instead of an external contract for this.